src/jmeter.ts

```diff
@@ -0,0 +1,214 @@
+const completionSpec: Fig.Spec = {
+  name: "jmeter",
+  description: "Apache JMeter - 100% Java Load Testing Tool",
+  options: [
+    {
+      name: ["-v", "--version"],
+      description: "print the JMeter version information and exit",
+    },
+    {
+      name: ["-h", "--help"],
+      description: "print usage information and exit",
+    },
+    {
+      name: ["-p", "--propfile"],
+      description: "the jmeter property file to use",
+      args: {
+        isOptional: false,
+        name: "property",
+        description: "Your jmeter property file",
+      },
+    },
+    {
+      name: ["-q", "--addprop"],
+      description: "additional JMeter property file(s)",
+      args: {
+        isOptional: false,
+        name: "addprop",
+        description: "additional JMeter property file(s)",
+      },
+    },
+    {
+      name: ["-t", "--testfile"],
+      description: "the jmeter test(.jmx) file to run. \"-t LAST\" will load last used file",
+      args: {
+        isOptional: false,
+        name: "testfile",
+        description: "the jmeter test(.jmx) file to run. \"-t LAST\" will load last used file ",
```

Comment:
Actually this could use the [`filepaths()`](https://fig.io/docs/reference/generator/prebuilt-generators#filepathsoptions) generator to allow specifying the file extension
```suggestion
        description: "the jmeter test(.jmx) file to run. \"-t LAST\" will load last used file ",
        template: "filepaths",
```